geoai/universat.py

```
@@ -0,0 +1,209 @@
+        [_proj(s, int(s.shape[0] ** 0.5) if s.ndim == 2 else s.shape[0]) for s in t]
+    )
+
+
+def universat_train(
+    experiment: str,
+    overrides: Optional[List[str]] = None,
+    project_root: Optional[str] = None,
+):
+    if not os.path.exists(p := os.path.expanduser("~/.cache/geoai/UniverSat")):
+        raise FileNotFoundError("UniverSat repo not found.")
+    subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)